app/src/main/java/com/ciandt/dragonfly/example/helpers/SizeHelper.kt

```diff
@@ -0,0 +1,14 @@
+package com.ciandt.dragonfly.example.helpers
+
+import java.text.DecimalFormat
+
+object SizeHelper {
+
+    fun toReadable(size: Long): String {
+        if (size <= 0) return "0"
+        val units = arrayOf("B", "kB", "MB", "GB", "TB", "EB")
```

Comment:
For internationalization, I think these labels should be passed as parameters, no?